tests/page/formHandler/formHandler.test.js

```diff
@@ -0,0 +1,192 @@
+import { expect } from 'chai';
+import { spy } from 'sinon';
+
+import { FORM_SUBMISSION_SUCCEEDED_TYPE, FORM_SUBMISSION_FAILED_TYPE } from '../../../src/actions.js';
+
+import Handler from '../../../src/page/formHandler/formHandler.js';
+
+const httpClientMocker = (success = true, response = {}) =>
```

Comment:
i think i see this in other test. Could it be move to one file and used via import ?